x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/EnterpriseSearch.java

```diff
@@ -90,17 +94,18 @@ public class EnterpriseSearch extends Plugin implements ActionPlugin, SystemInde
 
     public static final String BEHAVIORAL_ANALYTICS_API_ENDPOINT = APPLICATION_API_ENDPOINT + "/analytics";
 
+    public static final String QUERY_RULES_API_ENDPOINT = "_query_rules";
```

Comment:
Note: I purposely kept this external to the `_application` endpoint. (If we like this and think the classes need to move packages, I'd prefer to do that refactor/move in a single PR to limit noise)